plugins/providers/bigquery/model.go

```diff
@@ -62,11 +79,26 @@ func (t *Table) FromDomain(r *domain.Resource) error {
 }
 
 func (t *Table) ToDomain() *domain.Resource {
-	return &domain.Resource{
+	r := &domain.Resource{
 		Type: ResourceTypeTable,
 		Name: t.TableID,
 		URN:  fmt.Sprintf("%s:%s.%s", t.ProjectID, t.DatasetID, t.TableID),
 	}
+
+	var metadata map[string]interface{}
+	if t.Labels != nil {
+		metadata = map[string]interface{}{
+			"labels": t.Labels,
+		}
+	}
+	if metadata != nil {
```

Comment:
I didn't get this check. At this point, isn't Metadata always nil because of line 88? 🤔